php/containers.json

```diff
@@ -639,6 +641,29 @@
       "secrets": [
         "FULLTEXTSEARCH_PASSWORD"
       ]
+    },
+    {
+      "container_name": "nextcloud-aio-docker-socket-proxy",
+      "display_name": "Docker Socket Proxy",
+      "image": "nextcloud/aio-docker-socket-proxy",
+      "init": true,
+      "internal_port": "2375",
+      "environment": [
+        "TZ=%TIMEZONE%"
+      ],
+      "volumes": [
+        {
+          "source": "%WATCHTOWER_DOCKER_SOCKET_PATH%",
+          "destination": "/var/run/docker.sock",
```

Comment:
https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled

Reply:
Right, I forgot this. But I guess it would probably also produce problems with watchtower?

Reply:
I suppose we should probably set the flag for these containers then by default...

Reply:
maybe yes

Reply:
https://github.com/nextcloud/all-in-one/issues/3239